core/storage/feedback/gae_models.py

```diff
@@ -88,18 +97,39 @@ def generate_new_thread_id(cls, exploration_id):
 
     @classmethod
     def generate_full_thread_id(cls, exploration_id, thread_id):
+        """Generates a full thread ID which is unique globally
+
+        Args:
+            exploration_id: str,The exploration id the thread belongs to
+            thread_id: str, thread ID of given thread
+
+        Returns: full thread ID by concatenating exploration_id and thread_id
```

Comment:
This is not in the correct format. The type should be stated, followed by a full stop and a space, followed by the docstring. You do not need to explain the implementation, just the interface.